dlt/common/destination/capabilities.py

```diff
@@ -42,6 +42,7 @@ class DestinationCapabilitiesContext(ContainerInjectableContext):
     naming_convention: str = "snake_case"
     alter_add_multi_column: bool = True
     supports_truncate_command: bool = True
+    supports_numeric_precision_in_schema: bool = True
```

Comment:
add that to `add-a-destination.md` also I'm worried we inflate this capabilities so adding more destinations becomes harder

Reply:
Yes, you are right, I can also add this as some class var or internal setting that is not part of capabilities, not sure.